### PR TITLE
Fix Material#getTranslationKey for Block Materials

### PR DESCRIPTION
**Pull request by jpenilla**

### CODE REVIEW

File: Spigot-Server-Patches/0566-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch [hunk 1]
```diff
@@ -85,13 +85,16 @@ index 324afd2a5e50a3b1fd9de926f6da8836b00e173b..865c1e4cf18b25ecf2ba8d592d9ca40f
...
+@@ -393,6 +394,25 @@ public final class CraftMagicNumbers implements UnsafeValues {
              throw new RuntimeException();
          }
      }
 +
 +    @Override
 +    public String getTranslationKey(Material mat) {
++        if (mat.isBlock()) {
++            return getBlock(mat).getDescriptionId();
++        }
```
Comment (kashike) [inline Spigot-Server-Patches/0566-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch:97]: Missing start and end comments.

File: Spigot-API-Patches/0276-Cache-the-result-of-Material-isBlock.patch [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+     private final NamespacedKey key;
++    private final boolean isBlock = this.isBlock0(); // Paper
+ 
+     private Material(final int id) {
+         this(id, 64);
+@@ -3717,7 +3718,12 @@ public enum Material implements Keyed {
+      *
+      * @return true if this material is a block
+      */
++    // Paper start - cache isBlock
```
Comment (kashike) [inline Spigot-API-Patches/0276-Cache-the-result-of-Material-isBlock.patch]: This comment should be after the method start

**Review by kashike — CHANGES_REQUESTED** [review]

File: Spigot-Server-Patches/0566-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch (hunk 1 above)
Comment (jpenilla) [reply Spigot-Server-Patches/0566-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch:97]: this is code added by paper, there is a start comment many lines above and an end one many lines below

Comment (kashike) [reply Spigot-Server-Patches/0566-Add-a-way-to-get-translation-keys-for-blocks-entitie.patch:97]: ah ok

**Review by kashike — APPROVED** [review]

**Pull request merged**